packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -379,14 +379,15 @@ abstract class FlutterCommand extends Command<void> {
       );
   }
 
-  /// Adds options for connecting to the Dart VM observatory port.
+  /// Adds options for connecting to the Dart VM Service port.
   void usesPortOptions({ required bool verboseHelp }) {
-    argParser.addOption(observatoryPortOption,
+    argParser.addOption(vmServicePortOption,
+        aliases: const <String>['observatory-port'],
```

Comment:
Same comment, you can keep this alias around, or you can "deprecate" it and throw or warn when it's used.  @christopherfujino may have a preference.

Reply:
I'm ok with the alias, it's just another line of code and we won't break anyone's scripts.